src/main/java/org/broadinstitute/hellbender/utils/IndexUtils.java

```diff
@@ -25,8 +25,10 @@ private IndexUtils(){}
     /**
      * Load a Tribble .idx index from disk, checking for out of date indexes and old versions
      * @return an Index, or null if we're unable to load
+     *
+     * @param errorOnOutOfDateIndex If true, will raise a UserException when an out of date index file is detected.
      */
-    public static Index loadTribbleIndex(final File featureFile) {
+    public static Index loadTribbleIndex(final File featureFile, final boolean errorOnOutOfDateIndex) {
```

Comment:
On second thought, this doesn't seem to be very prevalent in the code so it is proably fine